src/server/req_delete.c

```diff
@@ -119,6 +120,27 @@ static char *sigtj =  SIG_TermJob;
 static char *acct_fmt = "requestor=%s@%s";
 static int qdel_mail = 1; /* true: sending mail */
 
+/**
+ * @brief
+ *     Wrapper function that calls process_hooks()
+ *
+ * @return	int
+ */
+
+int
+caller_to_process_hooks(struct batch_request *preq, char *hook_msg, size_t msg_len,
+	void	(*pyinter_func)(void))
+{
+	int rc;
+	rc = process_hooks(preq, hook_msg, msg_len, pyinter_func);
+	if (rc == -1)
+		log_event(PBSEVENT_DEBUG2, PBS_EVENTCLASS_HOOK,
+				LOG_INFO, "", "resv_end event: accept req by default");
```

Comment:
I think this log level is wrong. process_hook returns -1 when there is an internal error which is much more serious than hook rejecting the event. I think you should log PBSEVENT_ERROR here and DEBUG2 in its callers